ironfish/src/blockchain/blockchain.ts

```diff
@@ -1346,6 +1349,151 @@ export class Blockchain {
     }
   }
 
+  private async updateAssetsFromBlock(block: Block, tx: IDatabaseTransaction): Promise<void> {
+    for (const transaction of block.transactions) {
+      await this.processMintsToAssetsStore(transaction, tx)
+      await this.processBurnsToAssetsStore(transaction, tx)
+    }
+  }
+
+  private async processMintsToAssetsStore(
```

Comment:
This is a small nomenclature thing, but we use the term `connect` when were adding things to the main chain instead of `process`